builder/vmware/common/driver_esx5.go

```diff
@@ -857,7 +856,7 @@ func (d *ESX5Driver) VerifyChecksum(hash string, file string) bool {
 		Src: file + "?checksum=" + hash,
 	})
 	if err != nil {
-		log.Printf("coulnd't parse the checksum: %v", err)
+		log.Printf("Error parsing the checksum: %v", err)
```

Comment:
For logs I would suggest adding the `[ERROR]` prefix to the message

Reply:
Resolved in 67054cd.